qbittorrent_exporter/exporter.py

```diff
@@ -166,7 +168,8 @@ def main():
         "username": os.environ.get("QBITTORRENT_USER", ""),
         "password": os.environ.get("QBITTORRENT_PASS", ""),
         "exporter_port": int(os.environ.get("EXPORTER_PORT", "8000")),
-        "log_level": os.environ.get("EXPORTER_LOG_LEVEL", "INFO")
+        "log_level": os.environ.get("EXPORTER_LOG_LEVEL", "INFO"),
+        "include_uncategorized": os.environ.get("QBITTORRENT_INCLUDE_UNCATEGORIZED", "false"),
```

Comment:
This makes no sense anymore :smile:

Reply:
https://github.com/esanchezm/prometheus-qbittorrent-exporter/pull/1/commits/b4aefff06df2181ffd90e45c0190368852a7bed1
:P

Reply:
Sorry, maybe my comment wasn't clear enough. The changes to these lines are not needed anymore since including uncategorized torrents is the default setting. Hence, it doesn't make sense to add `include_uncategorized` to the config dictionary. It's also not used in all the code, so there's no reason to keep this. Hope it's clear now :smile:

Reply:
Ohhh my bad, I'll fix that tonight